pyproject.toml

```diff
@@ -42,8 +42,8 @@ include = [{ path = "migrations/sql/*", format = ["sdist", "wheel"] }]
 packages = [{ include = "task_processor"}]
 
 [tool.poetry.dependencies]
-python = "^3.10"
-django = "^3.2.23"
+python = ">=3.10,<4.0"
+django = ">=3.2.23"
```

Comment:
Just wondering if this should at least be <5?

Reply:
Probably not worth doing in my opinion?

Reply:
I think we can go without for now, but we might see unit test failures in the future if there are breaking changes added to future major django versions.

Reply:
The version is still locked, though, right? Unless someone updates the lock file, nothing should break?

Reply:
Sorry, yes, that's true 👍